src/midi_Defs.h

```diff
@@ -214,7 +214,7 @@ struct RPN
 #define MIDI_CREATE_INSTANCE(Type, SerialPort, Name)                            \
     midi::MidiInterface<Type> Name((Type&)SerialPort);
 
-#if defined(ARDUINO_SAM_DUE) || defined(USBCON)
+#if defined(ARDUINO_SAM_DUE) || defined(USBCON) || defined(__MK20DX128__) || defined(__MK20DX256__) || defined(__MKL26Z64__)
```

Comment:
I did not find any specific defines either (apart from `TEENSYDUINO=version`), but there's something more interesting: `SERIAL_PORT_HARDWARE_OPEN `, defined in `pins_arduino.h` (or `variant.h` on non-avr architectures). It seems to suited for our needs, see #65.